theme/ckeditor5-widget/widget.css

```diff
@@ -130,3 +130,9 @@
 		}
 	}
 }
+
+.ck-editor__editable.ck-read-only .ck-widget {
```

Comment:
Why not `.ck-editor__editable:not([contenteditable]) .ck-widget` instead? The [other pr](https://github.com/ckeditor/ckeditor5-ui/pull/453) won't be neccessary in such case.

Reply:
[After this comment](https://github.com/ckeditor/ckeditor5/issues/1261#issuecomment-424274233), I assumed it would be good idea (user-friendly) to introduce a dedicated class for readOnly mode like we have `ck-focused`, `ck-blurred`, `ck-disabled`,`ck-hidden` etc.

And IMH  it will be easier to handle `.ck-read-only` CSS instead of `*:not([contenteditable])` for basic users in the future.